infrastructure/bicep/parameters/prd.json

```diff
@@ -228,7 +228,7 @@
       "value": "https://eth-mainnet.g.alchemy.com/v2"
     },
     "zchfGraphUrl": {
-      "value": "https://api.thegraph.com/subgraphs/name/frankencoin-zchf/frankencoin-subgraph"
+      "value": "https://mainnetponder.frankencoin.com/"
```

Comment:
Is the `/` at the end right? Above there is no Slash at the end of the string

Reply:
Very interessing - just copied it from my .env setting and in my .env setting it is not with a slash at the end!?